Makefile

```diff
@@ -22,11 +22,10 @@ calicobuild.created: $(BUILD_FILES)
 	cd build_calicoctl; docker build -t calico/build-libnetwork .
 	touch calicobuild.created
 
-calicoctl:
-	#wget https://github.com/projectcalico/calico-docker/releases/download/v0.5.5/calicoctl
-	# Update to a release when there is one.
-	wget https://circle-artifacts.com/gh/projectcalico/calico-docker/1860/artifacts/0/home/ubuntu/calico-docker/dist/calicoctl
-	chmod +x calicoctl
+dist/calicoctl:
+	mkdir dist
+	curl -L https://github.com/projectcalico/calico-docker/releases/download/v0.10.0/calicoctl -o dist/calicoctl
```

Comment:
You can use the projectcalico link to get the latest calicoctl.  Don't we need to be using the latest versions to test everything together?